pkg/config/agent/config.go

```diff
@@ -204,6 +204,16 @@ type AgentConfig struct {
 	// NodeType is type of the Node where Antrea Agent is running.
 	// Defaults to "k8sNode". Valid values include "k8sNode", and "externalNode".
 	NodeType string `yaml:"nodeType,omitempty"`
+	// The expected Namespace in which the ExternalNode should be created for a VM or baremetal server Node.
+	// The default value is "default".
+	// It is used only when NodeType is externalNode.
+	ExternalNodeNamespace string `yaml:"externalNodeNamespace,omitempty"`
+	// The bypass policy rules are used to skip the check from Antrea NetworkPolicy rules.
+	// Each rule contains the following four items.
+	// direction (in|out), protocol(tcp/udp/icmp), remote IP or CIDR, dst port (ICMP doesn't require), and use ":" to
+	// split the items, e.g., "in:tcp:1.1.1.1:22"
+	// It is used only NodeType is externalNode.
+	BypassPolicyRules []string `yaml:"bypassPolicyRules,omitempty"`
```

Comment:
Should we name it "PolicyBypassRules" or "PolicyBypassingRules"

Reply:
Thanks, I have changed the field to "PolicyBypassRules".